components/org.wso2.carbon.identity.conditional.auth.functions.http/src/test/java/org/wso2/carbon/identity/conditional/auth/functions/http/HTTPGetFunctionImplTest.java

```diff
@@ -143,6 +144,21 @@ public void testHttpGetMethodWithHeaders() throws JsTestException {
         assertEquals(result, SUCCESS, "The http get request was not successful. Result from request: " + result);
     }
 
+    /**
+     * Test http get method with auth config.
+     * Check if the auth config is applied to the request.
+     *
+     * @throws JsTestException
+     */
+    @Test
+    public void testHttpGetMethodWithAuthConfig() throws JsTestException {
+
+        String requestUrl = getRequestUrl("dummy-get-with-auth-config");
+        String result = executeHttpGetFunction(requestUrl, TEST_AUTH_CONFIG);
+
+        assertEquals(result, SUCCESS, "The http get request was not successful. Result from request: " + result);
+    }
```

Comment:
Looks like this doesn't test the thing properly. Isn't the function includes, url, headers and the authconfig ?

Reply:
Yes it is.
Here TEST_AUTH_CONFIG [1] is considered the XML file for the test.

[1] https://github.com/wso2-extensions/identity-conditional-auth-functions/pull/157/files#diff-88e124ed9478132854f9f31dd8ae608c537a858165647f6d073ccb84c49e0646

Reply:
Let's have a chat on the unit tests separately. Will not consider the unit tests for the approval decision of this PR.